### ISSUE TITLE
Returning binary responses (image/png) possible?

### ISSUE THREAD

**Issue by aptfx** [anmonteiro/now-rust#1]
I tried this builder and it did work as expected. Now I thought about a simple example of generating a fractal png image within the rust lambda and returning the resulting image as binary data.

Its seems to be that this is not possible?

**Comment by anmonteiro** [comment]
Please point your issues to https://github.com/zeit/now-builders, that's where the builder is kept up to date. This was only a development version.

**Comment by anmonteiro** [comment]
@aptfx to answer your question though, it's possible to return binary data from the Now response.

**Comment by aptfx** [comment]
Ok... should I bring this discussion over there?

Reading from the code it seems to be base64 encoded? Can a now lambda based on rust get used to implement an "endpoint" that directly returns an image/png?

**Comment by anmonteiro** [comment]
> Ok... should I bring this discussion over there?
It's fine to discuss this one here..

> Reading from the code it seems to be base64 encoded? Can a now lambda based on rust get used to implement an "endpoint" that directly returns an image/png?

I think so? As long as you return a `Response<B>` in which `B` satisfies `Into<Body>`. Example trait implementation for `[u8]`: https://github.com/zeit/now-builders/blob/49e63de5fe08a135099fda317dff1f2ca5f851f6/packages/now-rust/src/body.rs#L156-L160

**Comment by aptfx** [comment]
It indeed seems to be quite easy... I'm just hitting the timeout limit to quickly :(. Even the hello world takes 12 minutes to build - my fractal times out at 15 minutes (as documented)

**Comment by anmonteiro** [comment]
Yeah that’s something that needs to be worked on before `@now/Rust` is announced.
For now you can use `debug: true` in config and that should get it to build given that debug builds are faster
